templates/svc/addons/rds-pg.yml

```diff
@@ -0,0 +1,105 @@
+Parameters:
```

Comment:
I am fully confused about when we do a service addon vs when we do an environment addon. 

@lgarvey Am I correct in thinking that this database will be destroyed if we delete the service?

Reply:
A service level addon is bound to the service, and will be nuked along with the service.  When we started work we only had service level addons.  Then they introduced env level addons.  

There's actually no facility to add a service level add on with `storage.yml` except for the s3 & s3-policy types which have a `services` key to add an identity policy in the service's addons directory.
 
We could remove most of the service level stuff to avoid confusion.

Reply:
Aren't these templates very similar? Can we make them moar generic and not separate them out into `env` and `svc` directories?

Is there a difference in the templates that I am missing?

Reply:
There's definitely something to do like that @yusufsheiqh . As well as the duplication angle, I think we want to be guiding people away from service addons to avoid exasperation when people accidentally delete a data store or something.